modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -10,11 +10,61 @@ Based on typical usage, the expected CPU utilization is less than 5%. For more i
 
```

Comment:
@sarathecoubian  Could you review this whole page? There are some minor edits that I think are needed. For example, the title `Anypoint Monitoring Performance, Impact, and Limits", and the subsequent 1 liner after that title, does not make sense. I think it requires just a brief paragraph to provide an overview what this page is about, but the current language doesn't make sense.

In addition, the section name `Data Retention and Aggregation` should be changed to `Data Retention and Limits`. The line below that says "...asychronously collect data..." doesn't make sense in this section. A better place for it would be in the "agent performance impact" section.

Reply:
Sure thing.

Reply:
Added. I struggled with that opener... Any ideas? I'd love to summarize everything that's in here without sounding so stiff.